.gitignore

```diff
@@ -156,3 +156,6 @@ $RECYCLE.BIN/
 .DS_Store
 /.vs/slnx.sqlite
 /.vs/Flow.Launcher.Plugin.Everything/DesignTimeBuild/.dtbcache.v2
+
+# Rider
+/.idea
```

Comment:
this is something required? (ok if it is)

Reply:
For rider config. Or else there will be some editor config files.